src/pswap.md

```diff
@@ -33,13 +33,13 @@ The 0.3% fee for every swap on the Polkaswap DEX is used to buy PSWAP tokens on
 
 ## PSWAP Token
 
-Polkaswap has 0.3% trading fees per transaction, which is the same as other contemporary exchanges like Uniswap. Unlike transaction fee models on other exchanges, on Polkaswap, trading fees are used to buy back and burn PSWAP tokens and then new PSWAP tokens are minted to reward LPs. Newly minted PSWAP tokens to liquidity providers start at 90% of the amount of burned PSWAP tokens in a 24 hour time period, and will gradually decrease down to a constant at 35% of daily burned tokens after 5 years. The PSWAP token model is thus deflationary in nature, unlike almost all other contemporary reward structures, because the rewards are limited by the amount burned in transaction fees each day.
+Polkaswap has 0.3% trading fees per transaction, which is the same as other contemporary exchanges like Uniswap. Unlike transaction fee models on other exchanges, on Polkaswap, trading fees are used to buyback-and-burn PSWAP tokens and then new PSWAP tokens are minted to reward LPs. Newly minted PSWAP tokens to liquidity providers start at 90% of the amount of burned PSWAP tokens in a 24 hour time period, and will gradually decrease down to a constant at 35% of daily burned tokens after 5 years. The PSWAP token model is thus deflationary in nature, unlike almost all other contemporary reward structures, because the rewards are limited by the amount burned in transaction fees each day.
```

Comment:
```suggestion
Polkaswap has a 0.3% trading fee per transaction, the same as other contemporary exchanges like Uniswap. Unlike transaction fee models on other exchanges, on Polkaswap, trading fees are used to buy back-and-burn PSWAP tokens, and then new PSWAP tokens are minted to reward LPs. Newly minted PSWAP tokens for liquidity providers start at 90% of the number of burned PSWAP tokens in 24 hours and will gradually decrease to a constant 35% of daily burned tokens after five years. The PSWAP token model is thus deflationary in nature, unlike almost all other contemporary reward structures, because the rewards are limited by the amount burned in transaction fees each day.
```